### PR TITLE
Create the execution id on the client side

### PR DESCRIPTION
**Pull request by plparent**
The execution id is created by the client and all files are uploaded from the client (except entrypoint.py since it is static).

### CODE REVIEW

File: frontend/src/components/ui/RunPipelineButton.jsx [hunk 1]
```diff
@@ -123,7 +47,14 @@ export default function RunPipelineButton({modalPopper, children, action}) {
     }
 
     let pipelineSpecs = editor.convert_drawflow_to_block(pipeline.name, pipeline.data);
-    pipelineSpecs = await processNodes(pipelineSpecs)
+    const executionId = uuidv7();
+    console.log(pipeline.buffer)
```
Comment (FGRCL) [inline frontend/src/components/ui/RunPipelineButton.jsx:51]: we can remove this debug log

File: main.go [hunk 2]
```diff
@@ -86,12 +88,12 @@ func createLogger(pipelineId string, file io.Writer, messageFunc func(string, st
 	return io.MultiWriter(os.Stdout, wsWriter, file)
 }
 
-
-func validateJson(ctx context.Context, body io.ReadCloser) (zjson.Pipeline, HTTPError) {
-	schema, err := json.Marshal(jsonschema.Reflect(&zjson.Pipeline{}))
+func validateJson[D any](ctx context.Context, body io.ReadCloser) (D, HTTPError) {
```
Comment (FGRCL) [inline main.go:91]: I made the changes here, but I'm not a go wizard. Does it look good to you?

File: executor.go [hunk 3]
```diff
@@ -629,34 +626,20 @@ func localExecute(pipeline *zjson.Pipeline, id int64, executionId string, cfg Co
...
 
-	executionFiles := s3key
-
 	eg, egCtx := errgroup.WithContext(ctx)
 	eg.SetLimit(runtime.NumCPU())
 
-	uploadedFiles := []string{}
 	for path, image := range blocks {
+		path := path
+		image := image
```
Comment (FGRCL) [inline executor.go:634]: What's the purpose of reassigning those values

Comment (plparent) [reply executor.go:634]: https://pkg.go.dev/golang.org/x/tools/go/analysis/passes/loopclosure

File: config.json [hunk 4]
```diff
@@ -13,6 +14,5 @@
 	"Local": {
 		"BucketPort": 8333,
 		"Driver": "docker-desktop"
-	},
-	"ZetaforgeIsDev": true
```
Comment (jmagoon) [inline config.json:17]: If this var is removed it needs to be updated in the analytics code as it's used for checking if it's a dev version or prod version

Comment (plparent) [reply config.json:17]: I simply renamed it to IsDev to be consistent with existing variable names

Comment (jmagoon) [reply config.json:17]: Other code refers to `ZetaforgeIsDev` so those sections need to be updated to read the new var name

Comment (plparent) [reply config.json:17]: I did not find any reference of `ZetarforgeIsDev` in the go code at least.

**Review by jmagoon — APPROVED** [review]

**Pull request merged**